test-apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,24 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+//I wasn't sure which pin to read from, so I assumed PA_0)
+CAN can2(PA_0, PA_1);
+
+/*
+Utilize the TutorialServo class that you wrote in the first part of the
+challenge. Use the same pin (PA_1) as used earlier in the training module. The program should
+be able to receive CAN messages which contain a percentage of the range of motion of the
+servo as a float.
+*/
+
+TutorialServo servo = new TutorialServo(PA_1, 180, 1, 2);
```

Comment:
No need for dynamic allocation, you can statically instantiate the servo object:
```
TutorialServo servo(PA_1, 180, 1, 2);
```

Reply:
Fixed in 0cba407156933737ef1cbe0b85f325a7b83d9f0a